scripts/deploy_glastopf.sh

```diff
@@ -36,8 +51,13 @@ phpize
 make && make install
 
 # Updated php.ini to add bfr.so
-BFR_BUILD_OUTPUT=`find /usr/lib/php5/ -type f -name "bfr.so" | awk -F"/" '{print $5}'`
-echo "zend_extension = /usr/lib/php5/$BFR_BUILD_OUTPUT/bfr.so" >> /etc/php5/apache2/php.ini
+if [ "$(lsb_release -r -s)" == "14.04" ]; then
```

Comment:
We warn user that 14.04 may not be supported, but here we have conditional installs dependent on 14.04. Are these installs a best effort and not a guaranteed fix?

Reply:
We warn that anything other than 14.04 or 16.04 are not supported. Here, we check if it's 14.04 and install the necessary bits. Otherwise, we install the necessary bits for 16.04. If it's something other than 14 or 16, it uses the 16.04 installation commands as those are the most up-to-date.